homeassistant/components/intesishome/climate.py

```diff
@@ -135,29 +146,50 @@ def __init__(self, ih_device_id, ih_device, controller):
         self._min_temp = None
         self._target_temp = None
         self._outdoor_temp = None
+        self._hvac_mode = None
+        self._preset = None
+        self._preset_list = [PRESET_ECO, PRESET_COMFORT, PRESET_BOOST]
```

Comment:
Should you also include a `PRESET_NONE` or must one always be selected ?

Reply:
One must always be selected in the only Intesishome devices I've been able to find which implement presets, thanks for checking!